contrib/cirrus/lib.sh

```diff
@@ -290,3 +290,42 @@ remove_packaged_podman_files() {
     # Be super extra sure and careful vs performant and completely safe
     sync && echo 3 > /proc/sys/vm/drop_caches || true
 }
+
+# Execute make localbenchmarks in $CIRRUS_WORKING_DIR/data
+# for preserving as a task artifact.
+localbenchmarks() {
+    local datadir
+    req_env_vars DISTRO_NV PODBIN_NAME PRIV_NAME TEST_ENVIRON TEST_FLAVOR
+    req_env_vars VM_IMAGE_NAME EC2_INST_TYPE
+
+    datadir=$CIRRUS_WORKING_DIR/data
+    mkdir -p $datadir
+
+    # Most of these are defined by cirrus and it's YAML.
+    # shellcheck disable=SC2154,SC2153
+    cat <<EOF | grep -Ev "$SECRET_ENV_RE" | sort > $datadir/benchmarks.env
```

Comment:
My request was `printenv | grep -Ev`, with the goal being to eliminate duplication. The grep added here is pretty much a NOP, and a misleading/confusing one.

(I don't think it's what's causing the build failure, though. I don't know what the problem is there).

Reply:
...okay, I think I got it now.